generate_parameter_library_py/generate_parameter_library_py/jinja_templates/cpp/parameter_library_header

```diff
@@ -182,6 +182,9 @@ struct StackParams {
 {%- endif %}
       updated_params.__stamp = clock_.now();
       update_internal_params(updated_params);
+      if (user_callback_) {
+         user_callback_(updated_params);
```

Comment:
The variable is called `update_params`. But I believe it contains all parameters, not just the ones that got updated right?

Would it be an idea to only supply the actually updated parameters? Otherwise the logging statements (like in the example) would be really verbose if you have multiple parameters. And from a log-reader standpoint you still don't have a clue what actually changed.

Reply:
Sure, can do!

One last question: is `setUserCallback` the right name here to differentiate these two use cases?
Because once this is in, it's harder to rename because of backward compatibility etc.